### DISCUSSION TITLE
RP 2040 programing in Visual Studio Code

### DISCUSSION THREAD

**Discussion by PragalfTJose** [microsoft/vscode-discussions#527] [Extension Development QnA]
Hai, 
I am working on a project based  on RP 2040 microcontroller. For this project I am trying to use Visual studio code for programing. I was  able to setup VS Code and build sample projects from pico sdk. Now I need to know how to create a new project using VS Code and build it. Also how to add header files to the main project.

Thank you.

**Comment by isidorn** [comment]
I suggest to ask this on StackOverflow or to google around. I am not sure. I am going to go ahead and close this since this is a community for extension authors.
